test/PriceOracleTest.js

```diff
@@ -0,0 +1,79 @@
+const { ethers, network } = require("hardhat");
+const { expect } = require("chai");
+
+const JXJOE_AGGREGATOR_ADDRESS = "0x02D35d3a8aC3e1626d3eE09A78Dd87286F5E8e3a";
+const PRICEORACLE_OLD_ADRESS = "0xe34309613B061545d42c4160ec4d64240b114482";
+
+const JXJOE_ADDRESS = "0xC146783a59807154F92084f9243eb139D58Da696";
+const XJOE_ADDRESS = "0x57319d41f71e81f3c65f2a47ca4e001ebafd4f33";
+const JOE_ADDRESS = "0x6e84a6216ea6dacc71ee8e6b0a5b7322eebc0fdd";
+
+describe("PriceOracleProxyUSD", function () {
+    before(async function () {
+        // Accounts
+        this.signers = await ethers.getSigners();
+        this.alice = this.signers[0];
+
+        // Contract Factory
+        this.priceOracleCF = await ethers.getContractFactory("PriceOracleProxyUSD")
+        this.JCollateralCapErc20Delegate = await ethers.getContractFactory("JCollateralCapErc20Delegate")
+        this.JErc20 = await ethers.getContractFactory("JErc20")
```

Comment:
Inconsistent naming here. 

First one is has lowercased first letter followed by `CF` at the end. The other two are uppercased first letter but don't have `CF` at the end.